apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -67,27 +71,36 @@ class DanceVisualizationColumn extends React.Component {
   };
 
   state = {
-    filterOff: this.setFilterStatus()
+    filterOn: this.initializeFilterStatus()
   };
 
   /*
     Turn the song filter off
   */
   turnFilterOff = () => {
-    this.setState({filterOff: true});
+    this.setState({filterOn: false});
   };
 
   /*
     The filter defaults to on. If the user is over 13 (identified via account or anon dialog), filter turns off.
    */
-  setFilterStatus() {
+  initializeFilterStatus() {
```

Comment:
A concern with this name is that it doesn't actually initialize anything, it just returns an initial value.  So I'd still suggest something more like `getInitialFilterStatus`.